src/test/java/net/masterthought/cucumber/json/EmbeddingTest.java

```diff
@@ -56,21 +49,15 @@ public static Iterable<Object[]> data() {
             { "video/mp4", "c29tZSBkYXRh", "some data", "embedding_-1003041823.mp4" },
         });
     }
-
-    @Parameter(0)
     public String mimeType;
-
-    @Parameter(1)
     public String data;
-
-    @Parameter(2)
     public String decodedData;
-
-    @Parameter(3)
     public String fileName;
 
-    @Test
-    public void getMimeType_ReturnsMimeType() {
+    @MethodSource("data")
+    @ParameterizedTest(name = "\"{0}\" with \"{1}\"")
+    public void getMimeType_ReturnsMimeType(String mimeType, String data, String decodedData, String fileName) {
```

Comment:
to be honest, older convention is more clear to read and understand

Reply:
I agree, this is harder to get used to but it was automated making it easier.  I've done these by hand and found them complicated in doing so.  I didn't check why the old copies were both ignored but I enabled them and they ran fine so nto sure if there was some issue.  You did have a issue link but it was just about adding the classes and didn't mention why they were set to ignored.

Reply:
Thank you for this upgrade - big effort !